### PR TITLE
docs: fix virtual cluster text that used double infobox

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR removes the double infobox introduced by PR #1598 .  @cloudmaniac in the future, avoid creating back-to-back info boxes. This is an anti-pattern due to how distracting they can be, and the text appears a bit too busy when info boxes are stacked. In your defense, we should have caught this before merging. 

CleanShot 2023-09-27 at 16 52 50

## Review Changes

💻 [Add Preview URL]()

### CODE REVIEW

File: docs/docs-content/clusters/cluster-groups/create-cluster-group.md [hunk 1]
```diff
@@ -8,9 +8,18 @@ tags: ["clusters", "cluster groups"]
 ---
 
 
-Use the instructions below to create a cluster group.
+Use a cluster group to organize your host clusters into a single logical group. A cluster group is a collection of one or more host clusters that together form a computing platform for you and your users to deploy Palette virtual clusters. Downstream consumers can use the cluster group when using Palette in [App Mode](../../introduction/palette-modes.md#what-is-app-mode).
```
Comment (ritawatson) [inline docs/docs-content/clusters/cluster-groups/create-cluster-group.md:11]: Do we need to make it "Palette Virtual Clusters" if this is a first occurrence?

File: docs/docs-content/clusters/cluster-groups/create-cluster-group.md [hunk 2]
```diff
@@ -53,11 +62,6 @@ s
 
 :::
 
-:::info
-
- Palette does not offer support for host clusters of these types within a cluster group: edge clusters, virtual clusters, Private Cloud Gateway (PCG) clusters, and imported clusters with read-only access.
-
```
Comment (ritawatson) [inline docs/docs-content/clusters/cluster-groups/create-cluster-group.md:59]: Judging by the line numbers, he wants to repeat this info here that also appears in lines 15-20. Is that right?

**Review by ritawatson — APPROVED** [review]
A couple questions. Otherwise LGTM.

File: docs/docs-content/clusters/cluster-groups/create-cluster-group.md (hunk 1 above)
Comment (karl-cardenas-coding) [reply docs/docs-content/clusters/cluster-groups/create-cluster-group.md:11]: Ah, great catch.

File: docs/docs-content/clusters/cluster-groups/create-cluster-group.md (hunk 2 above)
Comment (karl-cardenas-coding) [reply docs/docs-content/clusters/cluster-groups/create-cluster-group.md:59]: No, I moved the infobox content to the top, which is not in those lines you mentioned.

File: docs/docs-content/clusters/cluster-groups/create-cluster-group.md (hunk 1 above)
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/cluster-groups/create-cluster-group.md:11]:
```suggestion
Use a cluster group to organize your host clusters into a single logical group. A cluster group is a collection of one or more host clusters that together form a computing platform for you and your users to deploy Palette Virtual Clusters. Downstream consumers can use the cluster group when using Palette in [App Mode](../../introduction/palette-modes.md#what-is-app-mode).
```

**Pull request merged**